sched/semaphore/sem_rw.c

```diff
@@ -253,16 +255,13 @@ int init_rwsem(FAR rw_semaphore_t *rwsem)
 
   /* Initialize structure information */
 
-  ret = nxmutex_init(&rwsem->protected);
-  if (ret < 0)
-    {
-      return ret;
-    }
+#ifdef CONFIG_SMP
```

Comment:
let's add a dummy spin_lock_init in this case

Reply:
Done.